engine/class_modules/sc_death_knight.cpp

```diff
@@ -4998,6 +5007,10 @@ struct frost_strike_strike_t : public death_knight_melee_attack_t
     death_knight_melee_attack_t::execute();
 
     trigger_icecap( execute_state );
+
+    if ( p() -> conduits.unleashed_frenzy->ok() )
```

Comment:
Is this triggered on both strikes?

Reply:
Apparently it is.  I had someone on Archerus test it, and someone else confirmed it.  As odd as it sounds.

I was having issues trying to find any logs where someone was running both it, and dual wielding, so I wasn't able to confirm via that route.

At this point, in spelldata it has no ICD.  But eradicating blow does.

I'll be keeping an eye on this one as well.